ansible/roles/operators/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+# Openshift Virtualization channel
+osv_channel: 2.4
```

Comment:
Is this a value that will change with OCP release? If so should it be in role defaults? Also can't we use the channel like "4.5" etc like we do with other operators?

Reply:
Yes, it will change with Openshift release. This operator is maintaining its own channel number and this is latest.